src/Aspose.Imaging.Cloud.Sdk.Test/Aspose.Imaging.Cloud.Sdk.Test.csproj

```diff
@@ -33,6 +33,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="Aspose.Imaging" Version="19.12.0" />
```

Comment:
@dmitrypasyutin Is it necessary to reference Aspose Imaging package in the test project? Have you considered arranging test cases without the usage of this package?

Reply:
@s-zubov The use of Aspose.Imaging is justified in the comment above